x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/exchange/ExchangeServiceTests.java

```diff
@@ -300,41 +301,15 @@ Set<Integer> runConcurrentTest(
         int numSources = randomIntBetween(1, 8);
         List<Driver> drivers = new ArrayList<>(numSinks + numSources);
         for (int i = 0; i < numSinks; i++) {
-            String description = "sink-" + i;
```

Comment:
I don't spend much time down here (I think that's been mostly Nhat), but I think removing the description "sink-$i" may make debugging this harder if something fails. Maybe we want to preserve this one.

Same applies for preserving the description "source-$i" below.